README.md

```diff
@@ -310,7 +312,34 @@ Either `target` or `targetPercent` must be used.
   at which to run the queries. Default interpretation of unit if specified as a number
   in minutes.
 
----
+- **indicator** optional, represents the Service Level Indicator (SLI),
+  described in [SLI](#sli) section.
+  One of `indicator` or `indicatorRef` must be given in objective when creating composite SLO.
+
+- **indicatorRef** optional, this is the name of Service Level Indicator (SLI).
+  One of `indicator` or `indicatorRef` must be given when creating composite SLO.
+
+##### Notes (Composite SLO)
+
+**Composite SLO** goal of composite SLO is to enable user an end-to-end journey, it is done by defining many
+  independent objectives. Each objective can have different queries, data sources and targets. The basic implementation
+  assumes that the Composite Error Budget burns if the Error Budget for any of the SLO objectives within the Composite SLO
+  is burning. The logic of those calculation is the same for Composite SLOs as for regular (standard) objectives and SLOs.
+
+**Weight** allows the user to change the impact of a given SLO on the whole composite SLO. Weight is just multiplier, it
+  means that if weight is `0.5`, SLO will have half impact as default, on the other hand if weight is `100`, this SLO will
+  be 100 times more impactful. By default, weight has value 1 and doesn't need to be specified.
+
+**Calculations** should be as simple as possible to make composite SLO intuitive. Here is brief description how given budgeting
+  method should impact composite SLO and how wight scale its impact:
+
+- Occurrences - if SLO burns its budget composite is burning its budget at the same rate. Each violation that consumed
```

Comment:
Please add info that composite slo should use only one type of error budget calculation.

Reply:
Done